NETCORE/.vsts/linux-build.yml

```diff
@@ -25,31 +25,27 @@ steps:
 
 - task: DotNetCoreCLI@1
   displayName: Integration Tests
-  continueOnError: true
   inputs:
     command: "test"
     projects: "NETCORE/test/**/IntegrationTests.Tests.csproj"
     arguments: "--configuration Release -l trx"
 
 - task: DotNetCoreCLI@1
   displayName: Functional MVC Tests
-  continueOnError: true
   inputs:
     command: "test"
     projects: "NETCORE/test/**/*FunctionalTests.MVC.Tests.csproj"
     arguments: "--configuration Release --framework netcoreapp3.1 -l trx"
 
 - task: DotNetCoreCLI@1
   displayName: Unit Tests for AspNetCore 3.1
-  continueOnError: true
   inputs:
     command: "test"
     projects: "NETCORE/test/**/*AspNetCore.Tests.csproj"
     arguments: "--configuration Release --framework netcoreapp3.1 -l trx"
     
 - task: DotNetCoreCLI@1
   displayName: Unit Tests + Func Tests for WorkerService for AspNetCore 3.1
```

Comment:
will test failure in one framework cause other framework one to not run at all? We need to ensure they all run irrespective of failure of each other.

Reply:
at the moment, i don't see any other way to fail the build. :(

Reply:
I see.. If there is no other way, then lets merge this.

Reply:
Hey, i found the correct fix.
Please see my other PR: https://github.com/microsoft/ApplicationInsights-dotnet/pull/1892